.github/workflows/rocm-mi300.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+  pull_request:
```

Comment:
@amdfaa Please ensure this temporary change is reverted once the CI run for this workflow succeeds. We do not want to run these workflows on every PR.